R/select-helpers.R

```diff
@@ -107,27 +120,34 @@ num_range <- function(prefix, range, width = NULL, vars = peek_vars()) {
   match_vars(paste0(prefix, range), vars)
 }
 
-#' @export
 #' @rdname select_helpers
-#' @param ... One or more character vectors.
-one_of <- function(..., .vars = peek_vars()) {
-  keep <- compact(list(...))
+#' @param x A character vector.
+#' @inheritParams ellipsis::dots_empty
+#' @export
+all_of <- function(x, ..., vars = peek_vars()) {
```

Comment:
Why do we allow dots here? Or conversely, why don't we use them?

Reply:
They are not allowed because of the ellipsis check. They are forcing `vars()` to be passed by name.

Reply:
It doesn't seem useful to take data by dots. This is a way of being prudent and conservative with the function signature.

Reply:
:+1: can you make that clear in the news? (ie we’re being more conservative so the equivalent of `one_of(“a”, “b”)` is `all_of(c(“a”, “b”))`)

Reply:
Done in https://github.com/r-lib/tidyselect/commit/372eaf29f4244050d5e6a3b8308dddbccaa95247